### DISCUSSION TITLE
Does not detect Thermida in newer versions

### DISCUSSION THREAD

**Discussion by pka4916** [horsicq/Detect-It-Easy#113] [Q&A] [answered]
With DIE  2.0.5  it will detect Thermida in a file, which is correct.
But in DIE 3.01 / 3.0.4  is does not detect it as all..
What did change?

**Comment by horsicq** [comment]
Thanks a lot for the report. We will fix it.

**Comment by pka4916** [comment]
Was there any update on this?

Thank you

**Reply by horsicq** [reply]
Hello, I am working.

**Comment by BJNFNE** [comment, marked as answer]
@horsicq I was looking into it, For myself it was detecting an newly protected executable with Thermida correctly as Thermida/Winlicense

@pka4916 Can you may try again with the latest version to see if its working on your side aswell?
